docs/version-history/v24-version.hist

```diff
@@ -8,6 +8,10 @@
   or :ad-attr:`RequestDisk` might not match the slot created to run it.
   :jira:`3423`
 
+- Fixed a bug that would cause a crash in the *condor_rooster* and *condor_defrag* when many slots
```

Comment:
This sounds like it could use more word-smithing, but I can't actually come up with anything better.